File: JavaScript/0350_intersection_of_two_arrays_II/solution.js
function intersection_of_two_arrays(nums1,nums2) {
    let result = [];
    for (let num of nums1) {
        if (nums2.includes(num)) {
            result.push(num);
            let index = nums2.indexOf(num);
            nums2.splice(index,1);
        }
    }
    return result;
}

const nums1 = [4,9,5];
const nums2 = [9,4,9,8,4];
console.log(intersection_of_two_arrays(nums1,nums2))

File: Python/0350_intersection_of_two_arrays_II/solution.py
def intersection_of_two_arrays(nums1,nums2):
    result = []
    for num in nums1:
        if num in nums2:
            result.append(num)
            nums2.remove(num)
    return result

nums1 = [4,9,5]
nums2 = [9,4,9,8,4]
print(intersection_of_two_arrays(nums1,nums2))